lidar/README.md

```diff
@@ -0,0 +1,56 @@
+CEDAR-lidar
+---
+# Build package in docker
+
+1. Install Docker: https://docs.docker.com/get-docker/
+2. Install Docker compose: https://docs.docker.com/compose/install/ (for server)
+
+### Build docker image
+```bash
+cd lidar # Go to where Dockerfile is 
+docker build --tag lidar . # build the image
+ docker run --rm -it --entrypoint bash lidar 
+```
+# Build package locally
+
+### Install poetry
+1. install Python poetry: https://python-poetry.org/docs/#installation
+2. make sure your local python version is `3.8.6`. I recommand use `pyenv` to set up a local python version.
+```bash
+cd CEDAR/lidar
+poetry install --no-dev # install just the package for running (recommended)
+poetry install # install all packages
+```
```

Comment:
can all this install stuff be in a script?

Reply:
currently docker uses requirements.txt to install, since I think u use that as well?
so we can `-r` install it